docs/topics/api/overview.rst

```diff
@@ -377,6 +377,7 @@ v4 API changelog
 * 2020-09-17: dropped ``is_recommended`` from addons api - use ``promoted`` propety instead.  https://github.com/mozilla/addons-server/issues/15271
 * 2020-09-17: dropped ``recommended=true`` filter from addons api - use ``promoted=recommended`` filter instead.  https://github.com/mozilla/addons-server/issues/15467
 * 2020-09-17: added ``?promoted=badged`` search filter to addons api. https://github.com/mozilla/addons-server/issues/15468
+* 2020-10-08: changed reviewer submission subscriptions to accept listed and unlisted submissions. https://github.com/mozilla/addons-server/issues/15605
```

Comment:
According to [this page](https://addons-server.readthedocs.io/en/latest/topics/api/overview.html#api-versions-list), we shouldn't change URLs on v4, only requests/responses. Shouldn't we add the new endpoints to v5 instead? cc @eviljeff

Reply:
v4 is stable but not frozen, so we can change it (and do).  There is an argument that we shouldn't drop the existing `/subscribe/` endpoint though - it could be alias for `/subscribe_listed/`

Reply:
Ah, that makes sense. @eviljeff could you guide me how to do that alias without duplicating all the code? Can I just add a drf_action `subscribe` that calls `subscribe_listed`?

Reply:
> Ah, that makes sense. @eviljeff could you guide me how to do that alias without duplicating all the code? Can I just add a drf_action `subscribe` that calls `subscribe_listed`?

Yes, I think that'd work.

Reply:
Thanks! How do I declare that `subscribe` endpoint to be v4 only?

Reply:
> Thanks! How do I declare that `subscribe` endpoint to be v4 only?

you can't do it in the `@action`. I guess one way is to add the subscribe action to a subclass of  `AddonReviewerViewSet` and separate the v4 and v5 url_patterns like we do in addons/api_url.py.  There are probably other solutions.

Reply:
3e5a06c513b72cc1f4418dd012d7ce82904e8eea